src/ConstructiveSolidGeometry/LinePrimitives/LinePrimitives.jl

```diff
@@ -0,0 +1,3 @@
+struct LineSegments{T} <: AbstractLinePrimitive{T}
+    points::Vector{CartesianPoint{T}}
+end
```

Comment:
Do we really need this?

Shouldn't we rather implement broadcasting methods for `Vector{CartesianPoint{T}}`? :thinking:

Reply:
Good point! Let me check.